1-3.md

```diff
@@ -81,13 +85,14 @@ Approximately 30 to 45 minutes.
 
 #### Teaching Ideas
 
-* Discuss how accessibility solutions are also enjoyed by many people regardless of their disability. Emphasize how many solutions originally thought for accessibility have become mainstream, such as the type-writer, telephone, punch cards, text to speech, email, and voice controls.
-* Promote discussion among students about which aspects of accessibility benefit them as well as people with disabilities. For instance, ask them which products they can use without looking at them or without pressing any button.
+* Discuss how accessibility features benefit many people in different situations, such as mobile phone users, older people, and people using different hardware. Explain that many features originally thought for accessibility have become mainstream, such as the type-writer, telephone, punch cards, text to speech, email, and voice controls.
+* Promote discussion among students about which accessibility features they may already be using and which they may want to use. For example, ask them if they have used voice to dictate or to listen to messages.
```

Comment:
"benefit them as well as people with disabilities" implicitly assumes that students never have a disability themselves. Tried to make the wording more specific and clear